### PR TITLE
[zh] Translate declarative-config task

### PR DESCRIPTION
**Pull request by tengqm**
closes: #23086

### CODE REVIEW

**Review by markthink — COMMENTED** [review]
/lgtm
/approve

File: content/zh/docs/tasks/manage-kubernetes-objects/declarative-config.md [hunk 1]
```diff
@@ -0,0 +1,1553 @@
...
+<!--
+## Changing management methods
+
+Kubernetes objects should be managed using only one method at a time.
+Switching from one method to another is possible, but is a manual process.
+-->
+## 更改管理方法  {#changing-management-methods} 
+
+Kubernetes 对象在同一时刻应该只用一种方法来管理。
+从一种方法切换到另一种方法时可能的，但这一切换是一个手动过程。
```
Comment (SataQiu) [inline content/zh/docs/tasks/manage-kubernetes-objects/declarative-config.md:1424]:
```suggestion
从一种方法切换到另一种方法是可能的，但这一切换是一个手动过程。
```

**Review by SataQiu — COMMENTED** [review]
Thanks @tengqm !
/approve

File: content/zh/docs/tasks/manage-kubernetes-objects/declarative-config.md (hunk 1 above)
Comment (tengqm) [reply content/zh/docs/tasks/manage-kubernetes-objects/declarative-config.md]: done.

**Comment by SataQiu** [comment]
Thanks @tengqm 
/hold cancel
/lgtm

**Pull request merged**